packages/client/src/services/users.ts

```diff
@@ -30,9 +34,15 @@ const login = async (credentials: LoginCredentials) => {
   return response;
 };
 
+const update = async (id: string, partialUser: Partial<UserState>) => {
+  const response = await axios.put(`/users/${id}`, partialUser);
+  return response;
+};
+
 const authService = {
   signup,
   login,
+  update,
```

Comment:
Could we change this to updateUser please? 😊 And like maybe signUp so we stay consistent with Camel case? Smol ting but good ting y'knowmsayn?

Reply:
good point my brain fry, i will update update in the next task